src/components/Auth/AuthCheckProcessor.tsx

```diff
@@ -0,0 +1,170 @@
+            fingerprint: '',
+            model: '',
+            manufacturer: '',
+            brand: '',
+            product: '',
+            hardware: '',
+          },
+        },
+      },
+    }
+
+    //
+    // Part 5:  Make the Networking Call to Your Servers.  Below is just example code, you are free to customize based on how your own API works.
+    //
+    this.latestNetworkRequest = new XMLHttpRequest()
+    this.latestNetworkRequest.open('POST', `${Config.BaseURL}/auth_3d`)
+    this.latestNetworkRequest.setRequestHeader(
+      'Authorization',
+      `Bearer ${this.sdkToken}`
+    )
+    this.latestNetworkRequest.setRequestHeader(
+      'Content-Type',
+      'application/json'
+    )
+
+    this.latestNetworkRequest.onreadystatechange = () => {
+      //
+      // Part 6:  In our Sample, we evaluate a boolean response and treat true as success, false as "User Needs to Retry",
+      // and handle all other non-nominal responses by cancelling out.  You may have different paradigms in your own API and are free to customize based on these.
+      //
+
+      if (this.latestNetworkRequest.readyState === XMLHttpRequest.DONE) {
+        const responseJSON = JSON.parse(this.latestNetworkRequest.responseText)
+        try {
+          const responseObj = responseJSON
+
+          if (responseObj.success === true) {
+            this.success = true
+
+            faceScanResultCallback.succeed()
+            this.nextStep()
+          } else if (responseObj.success === false) {
+            this.success = false
+
+            console.log('User needs to retry, invoking retry.')
+            faceScanResultCallback.retry()
+          } else {
+            this.success = null
+
+            console.log('Unexpected API response, cancelling out.')
+            faceScanResultCallback.cancel()
+            this.nextStep()
+          }
+
+          this.events?.emit('complete', {
+            type: 'complete',
+            ...responseObj,
+          })
+          FaceTecSDK.unload(() => {})
+        } catch {
+          // CASE:  Parsing the response into JSON failed --> You define your own API contracts with yourself and may choose to do something different here based on the error.  Solid server-side code should ensure you don't get to this case.
+          this.success = null
+          const message =
+            'Exception while handling API response, cancelling out.'
+          this.events?.emit('error', { type: 'exception', message })
+          faceScanResultCallback.cancel()
+          this.nextStep()
+          FaceTecSDK.unload(() => {})
+        }
+      }
+    }
+
+    this.latestNetworkRequest.onerror = () => {
+      // CASE:  Network Request itself is erroring --> You define your own API contracts with yourself and may choose to do something different here based on the error.
+      const message = 'XHR error, cancelling.'
+      this.events?.emit('error', { type: 'exception', message })
+      console.log(message)
```

Comment:
Did you intend to leave this log here?

Reply:
Yes. We're logging all major events, which should include exceptions, session inits, etc. This `console.log` in specific is to keep consistent.